mzLib/MassSpectrometry/Deconvolution/Deconvoluter.cs

```diff
@@ -11,7 +11,7 @@ namespace MassSpectrometry
     public enum DeconvolutionType
     {
         ClassicDeconvolution,
-        AlexDeconvolution,
+        ExampleNewDeconvolution,
```

Comment:
Lets make this more informative

Reply:
I'd like my name to still be included tho